CMakeLists.txt

```diff
@@ -561,6 +561,12 @@ if (LLAMA_LTO)
     endif()
 endif()
 
+find_program(LLAMA_CCACHE_FOUND ccache)
+if (LLAMA_CCACHE_FOUND)
+    set_property(GLOBAL PROPERTY RULE_LAUNCH_COMPILE ccache)
+    set(ENV{CCACHE_SLOPPINESS} pch_defines,time_macros)
```

Comment:
It'd be best to leave out pch_defines since we don't use precompiled headers, and sloppy preprocessing in the future might lead to surprising issues.